### ISSUE TITLE
PhaBOX release via bioconda 

### ISSUE THREAD

**Issue by bernt-matthias** [KennthShang/PhaBOX#25]
I'm currently working on a release of phabox via bioconda. 

Was wondering why the title of the current release contains the text `" V1.1"` but the link `0.1.0` (plus beta): https://github.com/KennthShang/PhaBOX/releases/tag/v0.1.0-beta.

Would you consider this release as beta? Maybe then it would be better to stick with https://github.com/KennthShang/PhaTYP?

**Comment by bernt-matthias** [comment]
Made some progress here: https://github.com/bioconda/bioconda-recipes/pull/50753

Contains some updates to the setup.py and imports that you might want to include here.

**Comment by KennthShang** [comment]
Hi there,

Thanks for the effort! I have removed the v1.1 release. The v1.1 is for the module conflict and I have revised the codes so that the problem will no longer occur.

Best,
Jiayu

**Comment by bernt-matthias** [comment]
Thanks for the reply. Can you just make a new release? A release (with version) is needed for bioconda. 
As a comment: try to avoid to remove releases, people might refer to them, e.g. in publications.

**Comment by KennthShang** [comment]
Thanks for your suggestions. I will keep it in mind :)

Best,
Jiayu

**Comment by bernt-matthias** [comment]
FYI: A first version of phabox (version 2.0) is now in bioconda https://github.com/bioconda/bioconda-recipes/pull/50753

**Comment by KennthShang** [comment]
Hi, I found the recipes via the bioconda cannot correctly run PhaBOX 2. Because there is a large different between versions 1 and 2

Maybe you can use the latest `meta.yaml` and `pyproject.toml` in our GitHub folder to replace them? 

Best,
Jiayu

**Comment by bernt-matthias** [comment]
Dear @KennthShang I could work on this. Could you give me more information what does not work / what should be adapted? 

Which of the two files would be the source of truth: 

- https://github.com/KennthShang/PhaBOX/blob/main/meta.yaml
- https://github.com/KennthShang/PhaBOX/blob/main/phabox2.yaml

Note that you might also directly open PRs for the bioconda recipe on your own - you can ping me anytime.

Instead of the meta.yaml and phabox2.yaml in this repo (which you probably have for reproducibility) you could use the docker containers which are automatically built for every bioconda recipe: https://quay.io/repository/biocontainers/phabox?tab=info.

**Comment by KennthShang** [comment]
Hi @bernt-matthias 

The meta.yaml is the receipt I used for conda build (or mamba build). Same format as the bioconda receipt I suppose. 

Also, I built the latest Phabox package, which is available via my personal channel (as mentioned in the current WIKI).

When I tried the release via bioconda channel, some modules were not available, and I suppose they are the following ones:

- `Kcounter -c bioconda` (Used in contamination)
- `prodigal-gv -c bioconda `(Used to replace the prodigal version in phabox2, since currently it is generalized for all viruses
- `mafft -c bioconda` (Used for marker genes MSA
- `fasttree -c bioconda` (Used for phylogenetic tree construction
- `seaborn -c conda-forge` (Used for sub-network visualization)

Thank you very much for your valuable suggestions :). We are still maintaining our phabox webserver and other issues. I will try to release a docker container once we fix them all.

(I would like to pin this issue if you would not mind.

Best,
Jiayu
